db/migrate/20160728193153_add_index_to_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToPieces < ActiveRecord::Migration
+  def change
+  end
+  add_index :pieces, :game_id
```

Comment:
You'll want this line to be inside

```Ruby
def change
  add_index :pieces, :game_id
end
```

The `change` method makes it easy for that migration to be run/rolledback. If someone runs db:rollback on that migration. `change` will do the opposite of adding the index (remove it).